src/components/Accordion/_index.scss

```diff
@@ -4,8 +4,22 @@
 /// @copyright IBM Security 2018
 ////
 
+// Security variables
+@import '../../globals/variables/index';
+
+// Carbon components
 @import 'carbon-components/scss/components/accordion/accordion';
 
+// Security components
+@import '../../components/SkeletonText/index';
+
+// Carbon mixins
+@import '@carbon/type/scss/index';
```

Comment:
This is more of a general question, but should we be referencing the package entry points or the specific modules? In this case `@carbon/type/scss/styles`

Reply:
Good point - I guess we should reference the relevant partials only. Will update now!